src/Essentials/src/PublicAPI/net-android/PublicAPI.Shipped.txt

```diff
@@ -1294,20 +1293,6 @@ static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTime defau
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTime defaultValue) -> System.DateTime
 static Microsoft.Maui.Storage.Preferences.Remove(string! key, string? sharedName) -> void
 static Microsoft.Maui.Storage.Preferences.Remove(string! key) -> void
-static Microsoft.Maui.Storage.Preferences.Set(string! key, bool value, string? sharedName) -> void
```

Comment:
We can t remove these lines fro mth Shipped. 

We need to add to unshipped like so 
```
*REMOVE*static Microsoft.Maui.Storage.Preferences.Set(string! key, bool value, string? sharedName) -> void
```

Reply:
Sorry about that